### DISCUSSION TITLE
Error loading repo

### DISCUSSION THREAD

**Discussion by mpharand** [tsightler/ring-mqtt#909] [Support Q&A] [answered]
Good Day.  Love your repo, I've got it running solid on main house.  However I'm getting the following error when doing new load on new home.

Logger: homeassistant.components.hassio
Source: components/hassio/websocket_api.py:135
integration: Home Assistant Supervisor (documentation, issues)
First occurred: 7:40:02 AM (2 occurrences)
Last logged: 7:40:02 AM

Failed to to call /store/repositories - Cmd('git') failed due to: exit code(8) cmdline: git fsck stderr: 'error: invalid HEAD notice: No default references'
Failed to to call /store/repositories - Cmd('git') failed due to: exit code(128) cmdline: git clone -v --recursive --depth=1 --shallow-submodules -- https://github.com/tsightler/ring-mqtt-ha-addon /data/addons/git/03cabcc9 stderr: 'Cloning into '/data/addons/git/03cabcc9'... POST git-upload-pack (175 bytes) error: could not lock config file /data/addons/git/03cabcc9/.git/config: No such file or directory fatal: could not set 'core.repositoryformatversion' to '0' '

Thanks M

**Comment by tsightler** [comment, marked as answer]
Based on the fact that `git fsck` is failing that would indicate that the local git database is damaged, or the filesystem is corrupted.  This is reinforced by the fact that the attempt to pull the addon failed with unable to lock local file, which has nothing to do with the repo itself.

**Reply by mpharand** [reply]
changed router to unifi, all good now.
